frontend/src/components/form/base/__tests__/EColorPicker.spec.js

```diff
@@ -46,6 +46,11 @@ describe('An EColorPicker', () => {
 
   const mount = (options) => mountComponent(EColorPicker, { vuetify, i18n, ...options })
 
+  beforeAll(() => {
+    // prevent "[Vuetify] Unable to locate target [data-app]" warnings
+    document.body.setAttribute('data-app', 'true')
```

Comment:
Das kann man umgehen, indem man gleich im template \<div data-app\> setzt.
https://github.com/ecamp/ecamp3/pull/701/files#diff-7eacba905c66292c1fc86af66485b30bb2ee2c01a7106822dd9d48537ec3469cR24

Siehe auch
https://github.com/ecamp/ecamp3/wiki/Frontend#tests-for-components-in-srccomponentsformbase-prefixed-with-e